zulipterminal/ui_tools/boxes.py

```diff
@@ -1412,13 +1418,21 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                 recipient_user_ids=[self.message['sender_id']],
             )
         elif is_command_key('MENTION_REPLY', key):
-            self.keypress(size, 'enter')
-            mention = f"@**{self.message['sender_full_name']}** "
-            self.model.controller.view.write_box.msg_write_box.set_edit_text(
-                mention)
-            self.model.controller.view.write_box.msg_write_box.set_edit_pos(
-                len(mention))
-            self.model.controller.view.middle_column.set_focus('footer')
+            stream_id = cast(int, self.stream_id)
+            if(not self.model.is_user_in_unsubscribed_stream(stream_id)):
+                self.keypress(size, 'enter')
+                mention = f"@**{self.message['sender_full_name']}** "
+                controller_write_box = self.model.controller.view.write_box
+                controller_write_box.msg_write_box.set_edit_text(
+                    mention)
+                controller_write_box.msg_write_box.set_edit_pos(
+                    len(mention))
```

Comment:
This refactoring is different from the conditional you're adding; when you're making so many changes, keep them focused on the key intention of the commit.